docs/openai.md

```diff
@@ -0,0 +1,135 @@
+# OpenAI compatibility
+
+Ollama provides experimental compatibility with parts of the [OpenAI API](https://platform.openai.com/docs/api-reference) to help connect existing applications to Ollama.
+
+> **Note:** OpenAI compatibility is experimental and is subject to major adjustments including breaking changes. For fully-featured access to the Ollama API, see the Ollama [Python library](https://github.com/ollama/ollama-python), [JavaScript library](https://github.com/ollama/ollama-js) and [REST API](https://github.com/jmorganca/ollama/blob/main/docs/api.md).
+
+## Usage
+
+### OpenAI Python library
+
+```python
+from openai import OpenAI
+
+client = OpenAI(
+    # This is the default and can be omitted
+    base_url='http://localhost:11434/v1/',
+    api_key='ollama',
```

Comment:
Might be worth mentioning `api_key` needs to be set to a non-emtpy value but it's unused and `ollama` here is just an example

Reply:
What happens if it's empty? I'm assuming the openai library throws an error. Maybe we can report that as an upstream issue? :-D

Reply:
Left a comment!